server/.gitignore

```diff
@@ -1,2 +1,4 @@
 config.json
 *.pyc
+settings.json
```

Comment:
What is `settings.json`?

Reply:
oh god, I thought I'd asked it to ignore .gitignore on my side.  That's just my own teething issues, must have committed a .gitignore before I got it to ignore my .gitignore.  settings.json was created automatically by vscode for me when I switched the linting to flake and it looks like this:
`{
    "python.pythonPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Shared\\Python37_64\\python.exe",
    "python.linting.pylintEnabled": false,
    "python.linting.flake8Enabled": true,
    "python.linting.enabled": true
}`